src/nix/hash.cc

```diff
@@ -10,18 +10,21 @@ using namespace nix;
 struct CmdHash : Command
 {
     FileIngestionMethod mode;
-    Base base = SRI;
     bool truncate = false;
     HashType ht = htSHA256;
+    HashEncoding encoding = SRI;
     std::vector<std::string> paths;
     std::optional<std::string> modulus;
 
     CmdHash(FileIngestionMethod mode) : mode(mode)
     {
-        mkFlag(0, "sri", "print hash in SRI format", &base, SRI);
-        mkFlag(0, "base64", "print hash in base-64", &base, Base64);
-        mkFlag(0, "base32", "print hash in base-32 (Nix-specific)", &base, Base32);
-        mkFlag(0, "base16", "print hash in base-16", &base, Base16);
+        mkFlag(0, "base16", "print hash in base16", &encoding, Base16);
+        mkFlag(0, "base32", "print hash in base32 (Nix-specific)", &encoding, Base32);
+        mkFlag(0, "base64", "print hash in base64", &encoding, Base64);
+        mkFlag(0, "prefixed-base16", "print hash in base16", &encoding, PrefixedBase16);
+        mkFlag(0, "prefixed-base32", "print hash in base32 (Nix-specific)", &encoding, PrefixedBase32);
+        mkFlag(0, "prefixed-base64", "print hash in base64", &encoding, PrefixedBase64);
```

Comment:
Let's not add these `prefixed-*` options. This is a legacy format that we don't need to expose in the new UI.

Reply:
Ok!